Identity/DataAccessLayer/Repositories/UsersRepository.cs

```diff
@@ -22,14 +16,24 @@ public UsersRepository(DatabaseContext databaseContext)
         public async Task<User> GetByAccountnameAsync(string? accountName)
         {
             var user = await _databaseContext.Users.FirstOrDefaultAsync(x => x.AccountName == accountName);
-            if (user == null) { throw new NullReferenceException(); }
+
+            if (user == null) 
+            { 
```

Comment:
let it be for now, but in future microservices remove exceptions from repositories